### ISSUE TITLE
Had issue with http2

### ISSUE THREAD

**Issue by gpratt** [eyeseast/propublica-congress#29]
FYI I'm using python2 and I had an issue querying propublica using your http2 query.

I get an error of:

`SSLHandshakeError: [SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert handshake failure (_ssl.c:590)`

I fixed the error by switching the querying method to urllib2.  If other people are having the same issue as me happy to commit the change, otherwise if you've got a reason for using http2 that I'm not aware of we can keep it as is.

**Comment by eyeseast** [comment]
My main reason for httplib2 is because it comes with built-in caching, which is nice to have, especially since a lot of the data doesn't change often (and sometimes never, because it's historical). 

Are you getting the error on every query, or something in particular?

**Comment by gpratt** [comment]
I was getting the error on everything, including your example query, over
two different computers.  Its not just your code either, when I took the
relevant code block and ran it manually I got the same issue.

I think it might have to do with the way my python is configured, but not
quite sure what.  I got halfway into googling potential fixes before I
realized I could exactly duplicate the function with urllib2 and just used
those results instead.  If I get a chance I'll clean up my code, and submit
a pull request.  Probably not worth fixing, especially if no one else has
any issues with the package, but at least it would be worth having a
documented solution out there.

Gabriel Pratt
Bioinformatics Graduate Student, Yeo Lab
University of California San Diego
